File: .sdk/tm/csharp/test/ExistsTest.cs
// Hook0 SDK exists test.

using Xunit;

using Hook0Sdk;

namespace Hook0Sdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = Hook0SDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}

File: .sdk/tm/java/test/ExistsTest.java
package JAVAPACKAGE.sdktest;

import static org.junit.jupiter.api.Assertions.assertNotNull;

import org.junit.jupiter.api.Test;

import JAVAPACKAGE.core.Hook0SDK;

public class ExistsTest {

  @Test
  public void testMode() {
    Hook0SDK testsdk = Hook0SDK.testSDK();
    assertNotNull(testsdk, "expected non-nil SDK");
  }
}
